### DISCUSSION TITLE
Felhő alapú megoldás

### DISCUSSION THREAD

**Discussion by MicraMetripond** [nav-gov-hu/eRECEIPT#18] [Kérdések és válaszok]
Felhő alapú megoldás estében, a felhőszolgáltatónak illetve magának a tárhelynek (központi infrastruktúra), milyen követelményeknek kell megfelelni? Hogy történik az engedélyezés? Ki kínálhat felhőalapú megoldást?

**Comment by renced42** [comment]
Kedves @MicraMetripond 

Felhőszolgáltató a tervezetben nem szerepel így ilyen követelmény nem került megfogalmazásra.
